gitlabform/gitlabform/core.py

```diff
@@ -256,6 +257,37 @@ def process_merge_requests(self, project_and_group, configuration):
             logging.info("Setting approvers to users %s and groups %s" % (approvers, approver_groups))
             self.gl.put_approvers(project_and_group, approvers, approver_groups)
 
+    @if_in_config_and_not_skipped
+    @configuration_to_safe_dict
+    def process_members(self, project_and_group, configuration):
+        groups = configuration.get('members|groups')
+        if groups:
+            for group in groups:
+                logging.debug("Setting group '%s' as a member", group)
+                access = groups[group]['group_access'] if \
```

Comment:
`groups[group]['group_access']` has to be defined, according to https://docs.gitlab.com/ee/api/members.html#add-a-member-to-a-group-or-project . I would not introduce own arbitrary default.

Reply:
This functionality uses a different api endpoint (https://docs.gitlab.com/ee/api/projects.html#share-project-with-group) from the users, however I have grouped them somewhat logically in the configuration.

Reply:
Any chance of resolving this and merging?

Reply:
> Any chance of resolving this and merging?

Please see my comment above (I wrote it a while ago already but forgot to click _post_ , sorry).

Reply:
Only sending the parameter if it is defined now